tests/pages/firefox/features/feature.py

```diff
@@ -14,18 +14,24 @@ class FeaturePage(BasePage):
     _URL_TEMPLATE = '/{locale}/firefox/features/{slug}/'
 
     _download_button_locator = (By.ID, 'features-header-download')
-    _sticky_promo_modal_content_locator = (By.CSS_SELECTOR, '.mzp-c-sticky-promo.mzp-a-slide-in')
 
-    def wait_for_page_to_load(self):
-        el = self.find_element(By.TAG_NAME, 'html')
-        self.wait.until(lambda s: 'loaded' in el.get_attribute('class'))
+    # Used as a scroll target to move down the page, to trigger the sticky promo element
```

Comment:
Up to you - but this is a relatively low impact page. If you wanted to remove the test from here also I think that would probably be fine. Testing the promo on /firefox/ and /firefox/new/ should be enough coverage I think.

Reply:
Removing.